app/src/main/AndroidManifest.xml

```diff
@@ -3,7 +3,7 @@
   xmlns:tools="http://schemas.android.com/tools"
   package="net.squanchy">
 
-  <uses-permission android:name="android.permission.INTERNET" android:maxSdkVersion="22" />
+  <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
Come to think about it, it makes sense. The permission is just automatically granted as it's not dangerous, but it still exists